contract/test/test-vstoreShop.js

```diff
@@ -1,5 +1,5 @@
 // @ts-check
-import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { test as anyTest } from './prepare-test-env-ava.js';
```

Comment:
0e24ea3 is maybe only needed when `agoric install` is used. If I do `cd contract && yarn`, like mentioned in `CONTRIBUTING.md`, importing from `@agoric/zoe/tools` does not seem to generate the `AssertionError assert(refs.runnerChain) evaluated to a falsy value` error.

Reply:
seems harmless in any case